### PR TITLE
feature: bring back profile check

### PR DESCRIPTION
**Pull request by arlimus**
Check a profile; exit code 1 if the check finds any errors; 0 otherwise; print warnings

### CODE REVIEW

File: lib/vulcano/profile.rb [hunk 1]
```diff
@@ -41,10 +41,68 @@ def initialize(options = nil)
...
+        data[:impact] ||= 0.5
+        data[:impact] = 1.0 if data[:impact] > 1.0
+        data[:impact] = 0.0 if data[:impact] < 0.0
+        rules[id] = data
+      end
+      res['rules'] = rules
+      res
+    end
+
+    def check # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```
Comment (chris-rock) [inline lib/vulcano/profile.rb:70]: Can we add a short description of expected return values?

Comment (arlimus) [inline lib/vulcano/profile.rb:70]: Good point, thank you for pointing this out!

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by chris-rock** [comment]
thank you very much @arlimus

**Comment by chris-rock** [comment]
@delivery approve

**Comment by chef-delivery** [comment]
Change: 217ee46b-1a88-46a9-86a5-f65209c2d3eb approved by: @chris-rock

**Pull request merged**